### DISCUSSION TITLE
GitHub as Publishing Requirement

### DISCUSSION THREAD

**Discussion by JulianGro** [elementary/appcenter#2176] [Q&A]
Hello,
[the documentation](https://docs.elementary.io/develop/appcenter/publishing-requirements) lists a GitHub repository as a requirement for publishing to the AppCenter. Does this actually mean that other forges, such as Codeberg or GitLab, are not allowed?

**Comment by danirabbit** [comment]
:thinking: I think theoretically we should be able to pull a git repository from anywhere. That might be old where we had a bit more of an integration with github issues and automated testing in a previous version of the AppCenter Dashboard. It's worth trying to see if the build succeeds with hosts outside of github imo! If it fails for some reason, then I think what you could do is have a packaging repo on GitHub that only contains your flatpak manifest and your manifest can for sure point to code hosted anywhere
